Kiosk/SettingsHelper.cs

```diff
@@ -263,6 +264,29 @@ private async void LoadRoamingSettings()
                 this.TranslatorTextApiKey = value.ToString();
             }
 
+            value = ApplicationData.Current.RoamingSettings.Values["InkRecognizerApiKey"];
+            if (value != null)
+            {
+                this.InkRecognizerApiKey = value.ToString();
+            }
+
+            value = ApplicationData.Current.RoamingSettings.Values["InkRecognizerApiKeyEndpoint"];
+            if (value == null && ApplicationData.Current.RoamingSettings.Values["InkRecognizerApiKeyRegion"] != null)
+            {
+                var inkRecognizerApiRegion = ApplicationData.Current.RoamingSettings.Values["InkRecognizerApiKeyRegion"].ToString();
+                value = GetRegionEndpoint(inkRecognizerApiRegion);
```

Comment:
this is a little confusing.  GetRegionEndpoint already checks for the empty value so just do something like

if(value == null)
{
   value = GetRegionEndpoint(ApplicationData.Current.RoamingSettings.Values["InkRecognizerApiKeyRegion"].ToString())
}

Reply:
I agree with you suggested solution and that it gets confusing. I tried following what was done with the previous endpoint selection for the other apps within the Kiosk. Should I make this change to gain a little more clarity or leave it as is to keep it uniform with the rest of the file?